README.md

```diff
@@ -154,6 +154,36 @@ Kiev::Shoryuken.enable
 
 The name of the worker class is not logged by default. Configure [`persistent_log_fields` option](#persistent_log_fields) to include `"shoryuken_class"` if you want this.
 
+### AWS SNS
+
+To enhance messages published to SNS topics you can use the ContextInjector:
+
+```ruby
+sns_message = { topic_arn: "...",  message: "{...}" }
+Kiev::Kafka.inject_context(sns_message[:message_attributes])
+
+```
+
+After this operation the message attributes will also include required context for the Kiev logger.
+
+### Kafka
```

Comment:
Is this specific to a particular Kafka client? Might be a good idea to specify which one.

Reply:
Well it provides the headers for kafka messages as `Hash<String, String>`. Which are fine for all clients: `rdkafka-ruby` and `ruby-kafka`. So it's not gem-specific.